apps/jobs/views.py

```diff
@@ -624,6 +624,149 @@ def leaderboard(request, challenge_phase_split_id):
     return paginator.get_paginated_response(response_data)
 
 
+@api_view(["GET"])
+@throttle_classes([AnonRateThrottle])
+def complete_leaderboard(request, challenge_phase_split_id):
```

Comment:
Please add permissions so that only challenge host can view these entries -
```
@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
@authentication_classes((ExpiringTokenAuthentication,))
```

Reply:
`challenge_phase_split_id` -> `challenge_phase_split_pk`